server/segment_rpc.go

```diff
@@ -328,6 +330,18 @@ func SubmitSegment(sess *BroadcastSession, seg *stream.HLSSegment, nonce uint64)
 		return nil, err
 	}
 
+	ratPrice, _ := ratPriceInfo(sess.OrchestratorInfo.PriceInfo)
```

Comment:
Let's handle the error

Reply:
It's impossible for us to error on this call as if that would happen then we would already error out at `validatePrice`